docs/en/configuration.md

```diff
@@ -28,3 +28,31 @@ DMSDocumentAddController:
     - php
     - php5
 ```
+
+## Adding fields to the Query Builder ##
+Query builder fields are read from the DMSDocument::searchable_fields property set in [querybuilder.yml](../../_config/querybuilder.yml). Some default fields are provided and can be customised
+by modifying the field and/or filter properties of a field or adding a new field entirely. 
+
+[See here for information](https://docs.silverstripe.org/en/3/developer_guides/model/searchfilters/) on how to modify search filters and [see here for more information](https://docs.silverstripe.org/en/3/developer_guides/forms/field_types/common_subclasses/)
```

Comment:
If you use https://docs.silverstripe.org/en/developer_guides/model/searchfilters then it'll automatically redirect to the latest stable major version (3)